llvm/include/llvm/CodeGen/TargetInstrInfo.h

```diff
@@ -779,6 +795,10 @@ class TargetInstrInfo : public MCInstrInfo {
     /// Once this function is called, no other functions on this object are
     /// valid; the loop has been removed.
     virtual void disposed() = 0;
+
+    /// Return true if the target can expand pipelined schedule with modulo
+    /// variable expansion.
+    virtual bool isMVEExpanderSupported() = 0;
```

Comment:
Should be a sane default (false, almost certainly for now) to avoid forcing downstream targets to update their implementation.

Reply:
Fixed to provide a default that returns false.